TrafficCapture/dockerSolution/src/main/docker/migrationConsole/kafkaExport.sh

```diff
@@ -25,7 +27,9 @@ usage() {
   echo "Options:"
   echo "  --timeout-seconds                           Timeout for how long process will try to collect the Kafka records. Default is 60 seconds."
   echo "  --enable-s3                                 Option to store created archive on S3."
-  echo "  --s3-bucket-name                            Option to specify a given S3 bucket to store archive on".
+  echo "  --s3-bucket-name                            Option to specify a given S3 bucket to store archive on."
+  echo "  --partition-offsets                         Option to specify partition offsets in the format 'partition_id:offset,partition_id:offset'."
+  echo "  --partition-limits                          Option to specify partition limits in the format 'partition_id:num_records,partition_id:num_records'."
```

Comment:
Can you elaborate on how partition offsets and partition limits work together? If I provide both for a given partition_id, does that mean it will print messages from `offset`->`offset+num_records`? What if I only provide one of them?

Reply:
That's correct. I've updated the description to hopefully make that more clear.

When partition-offsets is not set for a partition, it defaults to the beginning of the partition in kafka